### PR TITLE
[Program: GCI] Prepare database table for task comments

### PR DESCRIPTION
**Pull request by daksha-aeer**
### Description
**"app/database/models/task_comment.py"** added to create database table for comments.

#### Attributes:
- **id**: ID of task comment(primary key)
- **task id**: ID of a task
- **user id**: ID of user commenting on task
- **creation time**
- **modification time**
- **comment**: A string storing comment made on task

**Create, Read, Update, Delete (CRUD)** functionality added.

### Type of Change:

- Code

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Unit tests
```sh
python -m unittest discover tests
```
task40_1

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by sara-02** [comment]
Hey @daksha-aeer is this a part of some GCI task?

**Comment by annabauza** [comment]
Yes it is @sara-02

**Review by annabauza — APPROVED** [review]

**Comment by sara-02** [comment]
Ok. I did not see any issue associated with PR, that is why I asked.

File: app/database/models/task_comment.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, unique=True)
+    user_id = db.Column(db.Integer, unique=True)
+    creation_time = db.Column(db.Float, nullable=False)
+    modification_time = db.Column(db.Float)
+    comment = db.Column(db.String, nullable=False)
+
+    def __init__(self, task_id, user_id, comment):
+        """"Initialises TaskCommentModel class with task_id, user_id, and comment. """
```
Comment (isabelcosta) [inline app/database/models/task_comment.py:28]:
```suggestion
        """Initialises TaskCommentModel class with task_id, user_id, and comment. """
```

**Comment by annabauza** [comment]
closed because this changes has been included in other PR

**Pull request closed without merging**